modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1339,6 +1339,27 @@ TEST_P(Test_ONNX_nets, Resnet34_kinetics)
     expectNoFallbacksFromIE(net);
 }
 
+TEST_P(Test_ONNX_nets, QuantStatic)
+{
+    if (backend != DNN_BACKEND_OPENCV || target != DNN_TARGET_CPU)
+        throw SkipTestException("Only the default backend; CPU target is supported");
```

Comment:
> default backend

default backend in builds with IE is IE, so this message should be corrected to be accurate.